File: dataset/human/c/file_2615.c
#include <stdio.h>

int main(){
  char s[4];
  int ans[3] = {};
  int i;
  scanf("%s",s);
  for(i = 0;i < 3;i++){
    switch(s[i]){
      case 'a':
        ans[0] = 1;
        break;
      case 'b':
        ans[1] = 1;
        break;
      case 'c':
        ans[2] = 1;
    }
  }
  if(ans[0] == 1 && ans[1] == 1 && ans[2] == 1){
    printf("Yes\n");
  }else{
    printf("No\n");
  }
  return(0);
}

File: dataset/human/java/file_2615.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		String s = sc.next();
		String t = sc.next();

		char[] cs = s.toCharArray();
		char[] ct = t.toCharArray();

		Arrays.sort(cs);
		Arrays.sort(ct);

		StringBuilder sbt = new StringBuilder();
		StringBuilder sbs = new StringBuilder();

		for (int i = 0; i < t.length(); i++) {
			sbt = sbt.append(ct[t.length() - 1 - i]);
		}
		for (int i = 0; i < s.length(); i++) {
			sbs = sbs.append(cs[i]);
		}

		String[] str = new String[2];
		str[0] = sbs.toString();
		str[1] = sbt.toString();

		Arrays.sort(str);

		String ans = "";
		if (sbs.toString().equals(sbt.toString())) {
			ans = "No";
		} else if (str[0].equals(sbs.toString())) {
			ans = "Yes";
		} else {
			ans = "No";
		}

		System.out.println(ans);
	}

}
